tests/test_penv.py

```
@@ -16,63 +16,71 @@
 import os
 import subprocess
 
-from pioinstaller import __version__, penv, python, util
+import pytest
 
+from pioinstaller import __version__, penv, python, util
```

Ruff (0.12.2)

`pioinstaller.python` imported but unused

Remove unused import: `pioinstaller.python`

(F401)